README.md

```diff
@@ -1,10 +1,10 @@
-# Git commit message
+# Git Commit Messages
```

Comment:
We prefer headline capitalization to be sentence-case over word-case. We do this for a bunch of reasons and to harmonize with Google's standards for blogs and pages.